### ISSUE TITLE
What about tools?

### ISSUE THREAD

**Issue by amerkoleci** [studio-nine/Nine.Graphics#6]
Probably not the correct place, but how do you plan to support tools? Would you create tools in WPF? Or you would think on some cross platform UI solution?

Thank you

**Comment by yufeih** [comment]
Yes, there will be a small portable UI library build on top of this graphics stack. Unlike WPF, it'll be minimum so don't expect dependency properties, data binding, styling, etc.

Tools will be based on this UI library. I am going to start out with 2D, so initially the tools will only target very basic 2D placements.

**Comment by amerkoleci** [comment]
Definetelly like the idea, maybe you can take look at this one: https://github.com/grokys/Perspex

**Comment by erictuvesson** [comment]
I have made a private example of the UI we have  in Engine Nine. It doesn't have any complex things, but I find that WPF is a bit to complex in some areas where it can be simple and still  powerful.

If we are considering using it, I will make it public when this project is "usable".

**Comment by game-libgdx-unity** [comment]
it would be best if nine engine could have the UI likes Unity or similarity!
